### PR TITLE
fix: #33 - Bug In Box Space And Bottom

### LINKED ISSUE #33: Bug In Box Space And Bottom  /  Error En El Espacio Del Cuadro Y La parte inferior.
# Explanation:

More space is needed between the box and the footer. It is desired that the modification be made and that the required space exists, so that when looking at the page it looks well presented. It is recommended to correct the scroll.

**Current result:** There is not much space between the box and the bottom of the page, There is scroll on the page.

**Expected result:** That there is more space between the box and the bottom of the page. There should be no scrolling on the page.

*Screen size:* Full screen.

## Screenshots:

Original: 
image

Our: 
image

## Checklist:
This issue can be closed when the following tasks are completed:
 
 - [ ] When the space between the box and the bottom of the page is as expected here.
 - [ ] When the scroll has been corrected.

## References:
 
Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down&page-id=0%3A1
Our: https://github.com/Azordev/search-by-time-github

----

# Spanish Version:

## Explicación:
Se necesita más espacio entre el cuadro y el pie de página. Se desea que se realice la modificación y que exista el espacio requerido, para que al mirar la página se vea bien presentada. Se recomienda corregir el scroll o desplazamiento.

**Resultado actual:** No hay mucho espacio entre el cuadro y la parte inferior de la página. Hay desplazamiento en la página.

**Resultado esperado:** Que haya más espacio entre el cuadro y la parte inferior de la página. No debería haber desplazamiento en la página.

*Tamaño de la pantalla:* Pantalla completa.

## Capturas de pantalla:
Original:
imagen

Nuestro:
imagen

## Lista de Verificación:
Este problema se puede cerrar cuando se completen las siguientes tareas:
 
 - [ ] Cuando el espacio entre el cuadro y la parte inferior de la página es el esperado aquí.
 - [ ] Cuando se haya corregido el scroll o desplazamiento.

## Referencias:

Original: https://www.figma.com/proto/9hPU5c4waRbBOd8msfEnKa/Search_by_time_github?node-id=2%3A8&scaling=scale-down&page-id=0%3A1
Nuestro: https://github.com/Azordev/search-by-time-github

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

- The block including github logo and form is centered with position absolute.

- The block itself has a maximum width to fit the design and be responsive.

- Unnecessary styles of the form block were removed.

- Removed the navbar logo on mobile for better visualization.

fixes #33 

#### Screenshots

_C__Users_eghin_Downloads_Azordev_search-by-time-github_index html(iPhone 12 Pro)
_C__Users_eghin_Downloads_Azordev_search-by-time-github_index html(Nest Hub Max)

#### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] When the space between the box and the bottom of the page is as expected here.
- [x] When the scroll has been corrected.

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -28,6 +28,13 @@ body {
 #app {
   height: 95vh;
 }
+.center-block {
+  left: 50%;
+  top: 45%;
+  transform: translate(-50%, -50%);
+  max-width: 526px;
+  width: calc(100% - 40px - 40px);
```
Comment (Israel-Laguan) [inline src/css/style.css:33]: Not sure about this one

File: src/css/style.css [hunk 2]
```diff
@@ -28,6 +28,13 @@ body {
 #app {
   height: 95vh;
 }
+.center-block {
+  left: 50%;
+  top: 45%;
+  transform: translate(-50%, -50%);
```
Comment (Israel-Laguan) [inline src/css/style.css]: Why not use flex?

File: src/css/style.css (hunk 1 above)
Comment (Eghinner) [reply src/css/style.css:33]: The calc is for this.

Captura de pantalla (1)

File: src/css/style.css (hunk 2 above)
Comment (Eghinner) [reply src/css/style.css]: For the top, for a little less distance from the top. it looks like it's not exactly centered.

File: src/css/style.css (hunk 1 above)
Comment (Israel-Laguan) [reply src/css/style.css:33]: What about using padding? also 40px don't apply on all devices, please note that in design it applies to iPhone but what about the other devices?

Comment (Eghinner) [reply src/css/style.css:33]: About the padding, the box-sizing property should solve this problem. And to make it work on all devices I will use percentages and media queries. Thanks for review

Comment (Eghinner) [reply src/css/style.css:33]: This is what it currently looks like. What do you think?
chrome_Gscefi1I7W

Comment (Israel-Laguan) [reply src/css/style.css:33]: Looks better for me! @victor-heliomar @angelik0828 what do you think?

**Review by angelik0828 — APPROVED** [review]

File: src/css/style.css (hunk 2 above)
Comment (heliomar-pena) [reply src/css/style.css]: @Eghinner Maybe you could use flex to center it, and then use position-relative or negative margin-top to fix that little distance from the top that you says.

File: src/css/style.css (hunk 1 above)
Comment (heliomar-pena) [reply src/css/style.css:33]: LGTM, good work

**Review by e-azocar — APPROVED** [review]

**Comment by e-azocar** [comment]
Please resolve conflicts

**Review by Israel-Laguan — APPROVED** [review]

File: src/css/style.css (hunk 2 above)
Comment (Eghinner) [reply src/css/style.css]: work in progress

Comment (Eghinner) [reply src/css/style.css]: @victor-heliomar  done

**Review by e-azocar — APPROVED** [review]

**Comment by e-azocar** [comment]
@Eghinner please resolve conflicts to merge

**Pull request merged**